### ISSUE TITLE
Empty Repo

### ISSUE THREAD

**Issue by mohsenuss91** [8-chems/SOM-Self-organizing-map#1]
No code found !

**Comment by 8-chems** [comment]
Yep, I am currently on Debian distribution, this repo will be re-organized once I get the development environment, thanks for the comment, you find in the master branch some code (it needs to be checked for its necessary libraries, I think there s a payable one that should be checked as well)
